cv_next/app/cv/[cvId]/components/commentSection/commentsSection.tsx

```diff
@@ -0,0 +1,33 @@
+"use client"
+
+import useSWR from "swr"
+import { fetchComments } from "@/app/actions/comments/fetchComments"
+import Comment from "./comment"
+import { useEffect, useState } from "react"
+import { useSupabase } from "@/hooks/supabase"
+
+export default function CommentsSection({ cv }: { cv: CvModel }) {
+  const { data: comments } = useSWR(cv.id, fetchComments)
+
+  const supabase = useSupabase()
+  const [userId, setUserId] = useState<string>("")
+
+  useEffect(() => {
+    async function getUser() {
+      const userId = (await supabase.auth.getUser()).data.user?.id
+      if (!userId) throw new Error("User not found") // TODO: handle this
```

Comment:
Here as well

Reply:
As far as I know this was done in the comments branch that improved several things here(including icons, responsiveness and more functionality)
I will make sure tho(or a least add it as a thing to handle there)

Reply:
Added a redirect to inactive if the user couldnt be found